### PR TITLE
Code Owners file

### PR DESCRIPTION
**Pull request by jas29**
For issue #1214 
Created code owners' file for the dev-hub project.

### CODE REVIEW

**Comment by jas29** [comment]
This is a very generic code owner's file and can be modified to add more owners and specify reviewers when pr is made for specific file types (eg- .js files).

Let me know who else should be added as code owner

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# The default owner of all the files in the repository (Global Owner) will be requested for review.
+*      @patricksimonian
+
```
Comment (patricksimonian) [inline CODEOWNERS]: let's add you in there as well as @sheaphillips and @ShellyXueHan :)

**Comment by patricksimonian** [comment]
this doesn't look rebased. does it need to be? @jas29

**Comment by jas29** [comment]
> this doesn't look rebased. does it need to be? @jas29

Multiple owners can be added within the same line.
Screen Shot 2020-01-13 at 2 42 53 PM

**Comment by patricksimonian** [comment]
I think this needs rebasing with master 🤔

**Comment by jas29** [comment]
> I think this needs rebasing with master 🤔 otherwise we will lose the journey map changes you made earlier today

This branch is now upto date with master

**Pull request merged**